src/webgpu/shader/validation/shader_io/util.ts

```diff
@@ -0,0 +1,78 @@
+/**
+ * Generate an entry point that uses an entry point IO variable.
+ *
+ * @param attribute The entry point IO attribute.
+ * @param type The type to use for the entry point IO variable.
+ * @param stage The shader stage.
+ * @param io An "in|out" string specifying whether the entry point IO is an input or an output.
+ * @param use_struct True to wrap the entry point IO in a struct.
```

Comment:
nit: These param docs may need to move to the members of the args object type below in order to get associated properly. Or maybe there's a tsdoc syntax for this, but iirc there may not be.

Reply:
Apparently there is syntax, which I've used. Don't really have a setup to check that it works though.